lib/stm32/f4/rcc.c

```diff
@@ -812,4 +935,9 @@ void rcc_clock_setup_hse_3v3(const struct rcc_clock_scale *clock)
 	rcc_osc_off(RCC_HSI);
 }
 
+void __attribute__((deprecated)) rcc_clock_setup_hse_3v3(const struct rcc_clock_scale *clock)
```

Comment:
yep :)

Reply:
Allow me to mark this as resolved. :innocent: